src/backend/tools/index.ts

```diff
@@ -750,6 +750,22 @@ function getVkd3dUrl(): string {
   return 'https://api.github.com/repos/Heroic-Games-Launcher/vkd3d-proton/releases/latest'
 }
 
+export function isVulkanSupported(): boolean {
+  if (!isLinux) {
+    return isMac
+  }
+
+  if (!any_gpu_supports_version([1, 1, 0])) {
+    logWarning(
+      'No GPU with Vulkan 1.1 support found, Vulkan will not work',
+      LogPrefix.DXVKInstaller
+    )
+    return false
+  }
+
+  return true
+}
```

Comment:
maybe this can be calculated once and put in a variable and then we just return that variable? so we don't print that warning every single time this method is called for the cases this is not supported